packages/cli/lib/artifact-generator.js

```diff
@@ -131,12 +134,22 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
     if (generationStatus) {
       await this._updateIndexFiles();
 
+      const classes = this.artifactInfo.name.split(this.classNameSeparator);
+      let classesOutput = '';
+
+      classes.forEach((c, i) => {
+        classesOutput +=
+          i + 1 === classes.length // if last element
+            ? utils.toClassName(c)
+            : utils.toClassName(c) + this.classNameSeparator;
+      });
```

Comment:
This can be simplified as follows:
```ts
const classesOutput = this.artifactInfo.name.split(this.classNameSeparator)
  .map(utils.toClassName)
  .join(this.classNameSeparator);
```